.github/workflows/merge-main.yml

```diff
@@ -97,7 +98,8 @@ jobs:
             parameters: -p URL=fom.nrs.gov.bc.ca
           - name: db
             file: db/openshift.deploy.yml
-            overwrite: false
+            overwrite: true
```

Comment:
I'm nervous about changing this overwrite configuration for prod.

Reply:
I think this is just let Derek's GitHub Action know we will pass some parameters, but I could try to set it to false.